input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
+        </tr>
+        <tr>
+            <td style="width: 25%;">Gender identity</td>
+            <td style="width: 25%;">US Core requires US Core Gender Identity extension. AU Core uses the International Gender Identity extension.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Patient.telecom</td>
+            <td style="width: 25%;">US Core requires all telecom to have system and value.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html">AU Core Practitioner</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitioner.html">US Core Practitioner</a></td>
+            <td style="width: 25%;">Practitioner.identifier</td>
+             <td style="width: 25%;">US Core requires minimum of 1, and all identifiers to have system and value.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html">AU Core PractitionerRole</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitionerrole.html">US Core PractitionerRole</a></td>
+            <td style="width: 25%;">PractitionerRole.telecom, PractitionerRole.endpoint</td>
+            <td style="width: 25%;">US Core requires telecom or endpoint to be present (pd-1).</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">PractitionerRole.telecom</td>
+            <td style="width: 25%;">US Core requires all telecom to have system and value.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" rowspan="3" style="width: 25%;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-smokingstatus.html">US Core Smoking Status Observation Profile</a></td>
+            <td style="width: 25%;">Observation.status</td>
+            <td style="width: 25%;">US Core requires status of 'final' or 'entered-in-error'.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.code</td>
+            <td style="width: 25%;">US Core extensible binding to <a href="https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1267.6/expansion">Smoking Status Type</a>.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.value[x]:valueCodeableConcept</td>
+            <td style="width: 25%;">US Core extensible binding to <a href="https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1267.3/expansion">Smoking status comprehensive</a>.</td>
         </tr>
     </tbody>
-</table>
\ No newline at end of file
+</table>
+
+##### Missing Data
```

Comment:
For the US Core Missing Data & Supressed Data items:
1. Replace "An AU Core compliant resource meets the requirements for Missing Data in the compared US Core profile(s)." with "AU Core compliant resources are compliant with US Core requirements for Missing Data."
2. Replace "contain any requirements on" with "does not include requirements for".

Reply:
Done